### ISSUE TITLE
Iteration Plan for April 2019

### ISSUE THREAD

**Issue by KexyBiscuit** [AOSC-Dev/aosc-os-abbs#1762]
This plan captures our work in **April**. The freeze date for this [season](https://gist.github.com/MingcongBai/78f6923828bccc51f81019fd99462e6a) has been deferred for 2 months, though DE related packages still won't be included since such limited time for the remaining work.

## Expected Updates

- [x] April 2, 2019: `msbuild` v16 [Visual Studio 2019 Launch Event](https://visualstudio.microsoft.com/vs2019-launch/) #1767
- [ ] April 2, 2019: KDE Plasma v5.15.4 [Schedules/Plasma 5](https://community.kde.org/Schedules/Plasma_5)
- [x] ~April 4, 2019: `vscode` v1.33 Microsoft/vscode#71059~
- [ ] :hand: April 9, 2019: Patch Tuesday for Microsoft [Security Update Guide](https://portal.msrc.microsoft.com/en-us/security-guidance)
  - [ ] .NET Core 2.1 v2.1.10 SDK v2.1.603 Runtime v2.1.10 [Release notes](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1.10/2.1.10.md)
  - [ ] .NET Core 2.2 v2.2.4 SDK v2.2.203 Runtime v2.2.4 [Release notes](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2.4/2.2.4.md)
- [x] April 10, 2019: `vscode` v1.33.1 Microsoft/vscode#72003 daa6e6a
- [ ] April 13, 2019: `plasma-framework` v5.57 [Schedules/Frameworks](https://community.kde.org/Schedules/Frameworks)
- [ ] April 15, 2019: `libreoffice` v6.2.3 [Release Plan 6.2](https://wiki.documentfoundation.org/ReleasePlan/6.2)
- [x] ~April 23, 2019: `nodejs` v12 [Releases](https://nodejs.org/en/about/releases/) #1817~
- [ ] April 29, 2019: `libreoffice` v6.1.6 [Release Plan 6.1](https://wiki.documentfoundation.org/ReleasePlan/6.1)

## Released Updates

- [x] March 8, 2019: `audacity` v2.3.1 [Audacity 2.3.1 Released](https://www.audacityteam.org/audacity-2-3-1-released/) #1787
- [x] March 20, 2019: `darktable` v2.6.2 [darktable 2.6.2 released](https://github.com/darktable-org/darktable/releases/tag/release-2.6.2) #1789
- [x] March 28, 2019: `powershell` v6.2.0 [v6.2.0 Release of PowerShell Core](https://github.com/PowerShell/PowerShell/releases/tag/v6.2.0) #1766
- [x] March 29, 2019: `typescript` v3.4.1 Microsoft/TypeScript#30281 [TypeScript 3.4.1](https://github.com/Microsoft/TypeScript/releases/tag/v3.4.1) dbf0958
- [x] ~April 4, 2019: `pgadmin` v4.4 [Version 4.4](https://www.pgadmin.org/docs/pgadmin4/dev/release_notes_4_4.html)~
- [x] ~April 5, 2019: `typescript` v3.4.2 [TypeScript 3.4.2](https://github.com/Microsoft/TypeScript/releases/tag/v3.4.2)~
- [x] April 8, 2019: `mono` v5.18.1.3 [Tarball](https://www.mono-project.com/download/vs/) #1791
- [x] April 9, 2019: `typescript` v3.4.3 [TypeScript 3.4.3](https://github.com/Microsoft/TypeScript/releases/tag/v3.4.3) bd64ccb
- [x] April 10, 2019: `pgadmin` v4.5 [Version 4.5](https://www.pgadmin.org/docs/pgadmin4/dev/release_notes_4_5.html) 36db3f9
- [x] April 11, 2019: `mono` v5.20.1.19 [Tarball](https://www.mono-project.com/download/stable/) #1795

## New Packages

- [ ] `dotnet-runtime`
  - [ ] Armv8-A 64-bit (AArch64)
  - [ ] Armv7-A
- [ ] `dotnet-sdk`
  - [ ] Armv8-A 64-bit (AArch64)
  - [ ] Armv7-A
- [ ] `powershell`
  - [ ] :hand: Armv8-A 64-bit (AArch64)
    - Blocked by PowerShell/PowerShell#9417
  - [ ] :hand: Armv7-A
    - Blocked by `openssl+1.0`

## Documentation

- [ ] More Lifecycle Policy
  - [ ] PowerShell
  - [ ] LibreOffice

## Deferred Items

- March 13, 2019: GNOME v3.32 [GNOME 3.32 Released](https://www.gnome.org/news/2019/03/gnome-3-32-released/)
- March 18, 2019: MATE v1.22 [
MATE 1.22 released](https://mate-desktop.org/blog/2019-03-18-mate-1-22-released/)
- April 18, 2019: KDE Applications v19.04 [Schedules/Applications/19.04 Release Schedule](https://community.kde.org/Schedules/Applications/19.04_Release_Schedule)

**Comment by KexyBiscuit** [comment]
Moving to [May](https://github.com/AOSC-Dev/aosc-os-abbs/issues/1830).
